Rivers of London/rivers-of-london.html

```diff
@@ -575,6 +591,116 @@ <h3>Equipment</h3>
     </div>
   </section>
 
+  <!------------------------- Translations? ------------------------->
+  <input type="hidden" name="attr_str_txt" value="Strength"/>
+  <input type="hidden" name="attr_con_txt" value="Constitution"/>
+  <input type="hidden" name="attr_dex_txt" value="Dexerity"/>
+  <input type="hidden" name="attr_int_txt" value="Intelligence"/>
+  <input type="hidden" name="attr_pow_txt" value="Power"/>
+  <input type="hidden" name="attr_luck_txt" value="Luck"/>
+
+  <input type="hidden" name="attr_athletics_txt" value="Athletics"/>
+  <input type="hidden" name="attr_drive_txt" value="Drive"/>
+  <input type="hidden" name="attr_navigate_txt" value="Navigate"/>
+  <input type="hidden" name="attr_observation_txt" value="Observation"/>
+  <input type="hidden" name="attr_read_person_txt" value="Read Person"/>
+  <input type="hidden" name="attr_research_txt" value="Research"/>
+  <input type="hidden" name="attr_sense_vestigia_txt" value="Sense Vestigia"/>
+  <input type="hidden" name="attr_social_txt" value="Social"/>
+  <input type="hidden" name="attr_stealth_txt" value="Stealth"/>
+
+  <!--  Roll Template
+    --  &{template:rol} {{name=Test}} {{success=[[50]]}} {{hard=[[floor(50/2)]]}} {{roll1=[[1d100]]}}
+    -->
+  <rolltemplate class="sheet-rolltemplate-rol">
+    <table>
+      <caption>{{name}} ({{success}}/{{hard}})</caption>
+      <tr style="background:#BEBEBE;">
```

Comment:
Can we use `divs` instead of a table? I realize this is copied over from an existing sheet, but Roll20 are actively discouraging using tables for layout: https://wiki.roll20.net/Designing_Character_Sheet_Layout#HTML_Table

Reply:
In fact, if you just convert them to `divs` with a reasonable class name, I can take care of the styling if you don't have time or inclination.

Reply:
short answer: absolutely. long answer: when I figure out how to style it :!